modules/form-field/react/stories/stories_Radio.tsx

```diff
@@ -143,4 +159,53 @@ storiesOf('Components|Inputs/Radio/React/Left Label/Radio', module)
         <Radio id="1" value="email" label="E-mail" />
       </ControlledComponentWrapper>
     </FormField>
+  ))
+  .add('Disabled', () => (
+    <FormField label="Label" labelPosition={FormField.LabelPosition.Left}>
+      <ControlledComponentWrapper
+        controlledProp={ControlledComponentWrapper.ControlledProp.Checked}
+      >
+        <Radio disabled={true} id="1" value="email" label="E-mail" />
+      </ControlledComponentWrapper>
+    </FormField>
+  ));
+
+storiesOf('Components|Inputs/Radio/React/Visual', module)
+  .addParameters({component: Radio})
+  .addDecorator(withReadme(README))
+  .add('States', () => (
+    <StaticStates>
+      <Table>
+        <thead>
+          <tr>
+            <th></th>
+            <th>No state</th>
+            <th>Hover</th>
+            <th>Focused</th>
+            <th>Active</th>
+          </tr>
+        </thead>
+        <tbody>
+          {[false, true].map(checked => {
```

Comment:
disabled? error states?

Reply:
I was wondering if we wanted to break them out differently like Nicholas had suggested, will fix!